core/string/ustring.cpp

```diff
@@ -1976,14 +1976,9 @@ Error String::parse_utf8(const char *p_utf8, int p_len, bool p_skip_cr) {
 	}
 }
 
-CharString String::utf8() const {
-	int l = length();
-	if (!l) {
-		return CharString();
-	}
-
-	const char32_t *d = &operator[](0);
-	int fl = 0;
+static int _utf8_length(const String &s) {
```

Comment:
```suggestion
static _FORCE_INLINE_  int _utf8_length(const String &s) {
```

I'm not sure if I see any reason to split it to the separate function, it's used only one.